### DISCUSSION TITLE
A way to list all rules?

### DISCUSSION THREAD

**Discussion by alkim0** [netblue30/firejail#6207] [Q&A] [answered]
Is there a way to list all the rules for a profile (preferably without having to run the underlying application)?

I would like something recursively expands all the `include` directives (ideally in a tree format), so that I can see all the rules for a profile without having to chase each include.

**Comment by ghost** [comment, marked as answer]
AFAIK nobody implemented such a feature yet. Great idea though!

**Reply by rusty-snake** [reply]
fjp had such a feature.

You can also implement this with a few lines python.
